src/codec/decodeMeasurement-delta-protobuf.js

```diff
@@ -14,5 +14,18 @@ module.exports = function(buffer){
     protoDecoded.forEach(function(measurement){
         measurement.signal_strengths = deltaDecode(measurement.signal_strengths);
     });
-    return protoDecoded;
+    return protoDecoded.map(function (measurement) {
+
+        var devices = measurement.signal_strengths.map(function (signal_strength, index) {
+            return {
+                signal_strength: signal_strength,
+                ID: (measurement.IDs && index < measurement.IDs.length) ? measurement.IDs[index] : undefined
```

Comment:
The decode part is made to be used server-side.

When this will be put in production server side, we don't know if we will be able to update every sensors at the same time (mostly because of connexion issues), so I thought it would be great if the server could receive data from old sensors. (And the price is just one check, so I don't think it's too much of a problem)